portfolio/src/main/java/com/google/sps/servlets/ImageAnalysis.java

```diff
@@ -0,0 +1,107 @@
+// package com.google.sps.servlets;
+
+// import com.google.cloud.storage.Blob;
+// import com.google.cloud.storage.BlobId;
+// import com.google.cloud.storage.BlobInfo;
+// import com.google.cloud.storage.Storage;
+// import com.google.cloud.storage.StorageOptions;
```

Comment:
Dead code (aka commented out) should never be committed. 

If you're experimenting with code and want to keep a reference for your own needs just move the code to its own branch. This allows you easily activate the experimental code, and avoid the headache of using // comments to toggle the functionality. Instead, simply hop between branches.